### PR TITLE
Add schema version property

### PR DESCRIPTION
**Pull request by geekscruff**

### CODE REVIEW

File: m3_json_schema.json [hunk 1]
```diff
@@ -5,11 +5,21 @@
...
     "properties"
   ],
   "properties": {
+
+    "schema_version": {
+      "$id": "#/properties/schema_version",
+      "type": "string",
+      "title": "Schema Version",
+      "description": "Version of the JSON Schema to validate against. Change this to make a new release.",
+      "pattern": "1.0.beta"
```
Comment (arwenhutt) [inline m3_json_schema.json]: @geekscruff just to check that I'm understanding, when we make a change to the schema, we would update the version number here (line 20)?  like: 
      "pattern": "1.1.beta"

File: m3.yml [hunk 2]
```diff
@@ -14,6 +14,8 @@
 #   type:  type of thing does the profile describe
 #   version:  version of the profile
 
+schema_version: "1.0.beta"
```
Comment (no-reply) [inline m3.yml]: what's the difference between this and the `version:` on [L15](https://github.com/samvera-labs/houndstooth/pull/17/files#diff-23771d2cb9e5c4238105ed88f20a8e9bL15)?

Comment (geekscruff) [reply m3.yml]: This is the schema/specification version, the other is for the specific profile, eg. UCSBs profile.

File: m3_json_schema.json (hunk 1 above)
Comment (geekscruff) [reply m3_json_schema.json]: Yep.

File: m3.yml (hunk 2 above)
Comment (no-reply) [reply m3.yml]: this is the version of M3? got it.

could we clarify that some way? i think "schema" is ambiguous.

Comment (geekscruff) [reply m3.yml]: m3_version? Plus a better description?

Comment (arwenhutt) [reply m3.yml]: I think that's a great idea.  m3_version is definitely less confusing.

**Pull request merged**